modules/xdcr-reference/pages/xdcr-filtering-data-type-conversion.adoc

```diff
@@ -0,0 +1,479 @@
+| The comparison `non-Null > Null` returns `true`.
+
+| 17
+| `Int`
+| `Boolean`
+| image:xdcr-reference:checkmark-green-sm.png[] Valid for all `Int` values.
+The `Int` value `0` is converted to the `Boolean` value `false`; all other `Int` values are converted to the `Boolean` value `true`.
+| The boolean-comparison `true > false` returns `true`.
+
+| 18
+| `Uint`
+| `Boolean`
+| image:xdcr-reference:checkmark-green-sm.png[] Valid for all `Uint` values.
+The `Uint` value `0` is converted to the `Boolean` value `false`; all other `Uint` values are converted to the `Boolean` value `true`.
+| The boolean-comparison `true > false` returns `true`.
+
+| 19
+| `Float`
+| `Boolean`
+| image:xdcr-reference:checkmark-green-sm.png[] Valid for all `Float` values.
+The `Float` value `0.0` is converted to the `Boolean` value `false`; all other `Float` values are converted to the `Boolean` value `true`.
+| The boolean-comparison `true > false` returns `true`.
+
+| 20
+| `String`
+| `Boolean`
+| image:xdcr-reference:checkmark-orange-sm.png[] Valid if `String` is case-insensitive `"true"` or `"false"`; in which case `String` is converted to its `Boolean` equivalent.
+| The boolean-comparison `true > false` returns `true`.
+
+| 21
+| `<Any-Except-Array>`
+| `Array`
+| image:xdcr-reference:cross-mark-red-sm-wide.png[] Invalid.
+| Collation comparison is performed.
+
+| 22
+| `<Any-Except-Object>`
+| `Object`
+| image:xdcr-reference:cross-mark-red-sm-wide.png[] Invalid.
+| Collation comparison is performed. is used.
+
+| 23
+| `<Any-Except-Time-and-String`
+| `Time`
+| image:xdcr-reference:cross-mark-red-sm-wide.png[] Invalid.
+| Collation comparison is performed.
+
+| 24
+| `String`
+| `Time`
+| image:xdcr-reference:checkmark-orange-sm.png[] Valid if `String` can be parsed as a parameter to the `DATE` function.
+| Standard comparision is performed if valid; otherwise, collation comparison is performed.
+
+| 25
+| `Array`
+| `Array`
+| image:xdcr-reference:dash-black-inline.png[] No conversion need be performed.
+| Comparison is performed along the following lines: if the length of `Array_1` is greater than the length of `Array_2`, then `Array_1 > Array_2` returns `true`.
+If the arrays are of equal length, corresponding objects are compared in sequence, until one is found to be the greater, and `true` is returned &#8212; _i.e._ `Array_1[object_1] > Array_2[object_1]`, `Array_1[object_2] > Array_2[object_2]`, and so forth.
+
+| 26
+| `Object`
+| `Object`
+| image:xdcr-reference:dash-black-inline.png[] No conversion need be performed.
+| Comparison is performed along the following lines: if the length of `Object_1` is greater than the length of `Object_2`, then `Object_1 > Object_2` returns `true`.
+If the objects are of equal length, corresponding bytes of the objects' data are compared in sequence, until one is found to be the greater, and `true` is returned &#8212; _i.e._ `Object_1[byte_1] > Object_2[byte_1]`, `Object_1[byte_2] > Object_2[byte_2]`, and so forth.
```

Comment:
Same here as above.

Reply:
See the solution I've attempted. I've broken the explanations out as notes beneath the table.